lib/breadcrumbs_on_rails/railtie.rb

```diff
@@ -10,7 +10,10 @@ module BreadcrumbsOnRails
 
   class Railtie < Rails::Railtie
     ActiveSupport.on_load(:action_controller) do
-      include BreadcrumbsOnRails::ActionController
+      # There is no need to include breadcrumbs on an API controller
+      unless self.name == 'ActionController::API'
```

Comment:
I think this is a much better solution, because in this case, nothing of BreadcrumbsOnRails is injected into the API controller.  I can't think of a use case for an API to have breadcrumbs.  An alternative implementation could involve checking `Rails.version >= '5.0.0'` or checking for `self.name == ActionController::API rescue NameError`.  Both of those seem uglier to me.